code/game/objects/structures/survey_probe.dm

```diff
@@ -134,3 +152,66 @@
 		timer_id = null
 		icon_state = "surveying_probe_deployed"
 
+// Language-specific probe versions for mapping.
+/obj/structure/survey_probe/sol
+	desc_extended = "\
```

Comment:
instead of copying the desc_extended for every one and adding a tidbit at the end, just add it in initialize, or add a new variable that adds it in the parent's initialize